i18n/en.json

```diff
@@ -836,7 +836,7 @@
 	"smw-datavalue-constraint-violation-must-exists": "Property \"$1\" has a <code>must_exists</code> constraint and value ''$2'' is violating that requirement.",
 	"smw-datavalue-constraint-violation-single-value": "The \"[[Property:$1|$1]]\" property has a <code>single_value</code> constraint and value \"$2\" is violating that requirement.",
 	"smw-constraint-violation-uniqueness": "A <code>unique_value_constraint</code> constraint is assigned to the \"[[Property:$1|$1]]\" property which only permits unique value assignments and the ''$2'' value annotation was already found to be annotated in the \"$3\" subject.",
-	"smw-constraint-violation-uniqueness-isknown": "A <code>unique_value_constraint</code> constraint is assigned to the \"[[Property:$1|$1]]\" property hereby only permits unique value annotations, ''$2'' already contains an annotated value with \"$3\" violating the uniqueness constraint for the current subject.",
+	"smw-constraint-violation-uniqueness-isknown": "A <code>unique_value_constraint</code> constraint is assigned to the \"[[Property:$1|$1]]\" property, therefore unique value annotations are permitted. ''$2'' already contains an annotated value with \"$3\", violating the uniqueness constraint for the current subject.",
```

Comment:
I think the word "only" should be put back here, otherwise the sentence no longer explicitly excludes non-unique values.

Reply:
My bad. Sending a fix.

Reply:
#5445